### PR TITLE
fix search response

### PR DESCRIPTION
**Pull request by 2403905**
## Description

We fixed the search response code from 500 to 400 when the request is invalid

## Related Issue

- Fixes [https://github.com/owncloud/ocis/issues/7812](https://github.com/owncloud/ocis/issues/7812)

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by micbar** [comment]
@2403905 @fschade Did this get stuck?

**Comment by fschade** [comment]
I missed the review request, sorry for that, I’ll check it in the morning, thanks for the reminder!

File: changelog/unreleased/fix-search-responce.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Bugfix: Fix search response
+
+We fixed the search response code when the request is invalid
```
Comment (fschade) [inline changelog/unreleased/fix-search-responce.md:3]: please add some more context what invalid means

**Review by fschade — APPROVED** [review]
one little thing in the changelog, expect that, LGTM 🎸

**Comment by 2403905** [comment]
> @2403905 @fschade Did this get stuck?

Yes, it got stuck but should be merged.

**Pull request merged**